src/banded/dot.jl

```diff
@@ -0,0 +1,43 @@
+function _dot(x::AbstractVector, A::BandedMatrix, y::AbstractVector)
+    require_one_based_indexing(x, y)
+    (axes(x)..., axes(y)...) == axes(A) || throw(DimensionMismatch())
```

Comment:
Add a comment `# This should include a message but matches LinearAlgebra.dot` to clarify why it looks incomplete

Reply:
👍